### PR TITLE
Allow people to donate using PayPal

### LINKED ISSUE #40: Implement "Donate" buttons
On [Support Us](https://na-ovoce.cz/en/support-us/) page, there will be buttons for small donations. One through GoPay and one through PayPal. Amount will be optional, but higher than 200 CZK.

### PR DESCRIPTION
**Pull request by stlk**
Resolves #40 
- [x] Working implementation
- [ ] Discuss user flow
- [ ] When user is signed in link donation to him

### CODE REVIEW

File: src/donate/paypal.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import paypalrestsdk
```
Comment (jsmesami) [inline src/donate/paypal.py:1]: Please add this to requirements.txt

File: src/donate/paypal.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+from .models import PayPalResponse
+
+class PayPal(object):
+
+    def __init__(self, client_id: str, client_secret: str, development: bool):
```
Comment (jsmesami) [inline src/donate/paypal.py:7]: Wow! First use of type annotations I have observed outside tutorials:)

File: src/locale/cs/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -7,7 +7,7 @@ msgid ""
 msgstr ""
```
Comment (jsmesami) [inline src/locale/cs/LC_MESSAGES/django.po:7]: Don't bother with translations for now, they often cause merge conflicts and we'll tune them later anyway. But thanks for being exhaustive:)

File: src/donate/forms.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import django.forms
+from django.core.validators import MinValueValidator
+from django.utils.translation import ugettext_lazy as _
+
+MIN_DONATION_VALUE = 200
+
+class DonateForm(django.forms.Form):
```
Comment (jsmesami) [inline src/donate/forms.py:7]: Please use pep8 linter. I would format this as follows:

```
amount = django.forms.DecimalField(
    initial=MIN_DONATION_VALUE,
    validators=[MinValueValidator(
        MIN_DONATION_VALUE, 
        message=_('We accept only donations higher than {0} CZK').format(MIN_DONATION_VALUE),
    )]
)
```

or

```
amount = django.forms.DecimalField(
    initial=MIN_DONATION_VALUE,
    validators=[
        MinValueValidator(
            MIN_DONATION_VALUE,
            message=_('We accept only donations higher than {0} CZK').format(MIN_DONATION_VALUE),
        ),
    ]
)
```

**Comment by jsmesami** [comment]
I'm really sorry, but there has been pressure to implement this with [a partner](http://www.nadacevia.cz/): https://na-ovoce.cz/cs/podporte-nas/financne/

**Comment by stlk** [comment]
No problem. It looks great btw :)

**Pull request closed without merging**